org-code-javabuilder/neighborhood/build.gradle

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
is the reason this is within `org-code-javabuilder` is this script builds neighborhood specifically for javabuilder?

Reply:
Yes. The different parts are loosely interdependent. Using gradle sub-projects, we can still publish specific packages from here, but it's much easier to import shared utilities which we'll need to do in order to manage passing signals from the mini-apps to Javalab.